_config.yml

```diff
@@ -1 +1 @@
-baseurl: ""
\ No newline at end of file
+baseurl: ""
```

Comment:
is this required?

Reply:
Actually it got edited accidentally and then I changed it back to its original state. I guess I mistakenly let this line remain. So this is not required. Should I just remove that quickly?

Reply:
Changed that

Reply:
Oh, sorry.
That line (baseurl: "") is a part of the YML config file in the main repository. So I guess it should remain.